apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -2598,9 +2599,19 @@ suite('conversation.js >', function() {
     setup(function(done) {
       ConversationView.initializeRendering();
       setActiveThread(8);
-      container =
-        ConversationView.getMessageContainer(testMessages[0].timestamp, false);
+
+      container = ConversationView.getMessageContainer(
+        testMessages[0].timestamp, false
+      );
+
+      this.sinon.stub(MessageManager, 'getMessage').returns(Promise.resolve());
+
+      this.sinon.stub(Threads, 'unregisterMessage', (messageId) => {
+        ConversationView.activeThread.messages.delete(messageId);
+      });
```

Comment:
can you explain why this is needed, instead of simply checking whether the stub is called ?